src/saturn_engine/worker_manager/api/jobs.py

```diff
@@ -93,3 +100,35 @@ def post_fetch_states() -> Json[FetchCursorsStatesResponse]:
             session=session,
         )
     return jsonify(FetchCursorsStatesResponse(cursors=cursors))
+
+
+@bp.route("/_start", methods=("POST",))
+def post_start_job() -> Json[JobsStartResponse]:
+    restart: bool = flask.request.args.get("_restart", "false") == "true"
+    start_input = marshall_request(StartJobInput)
+    if not start_input.name and not start_input.job_definition_name:
```

Comment:
nit, would make sense to make it mutually exclusive -> `bool(start_input.name) != bool(start_input.job_definition_name)`, and make the error `MUST_SPECIFY_ONE_JOB`?